lib/blueprinter/extractors/association_extractor.rb

```diff
@@ -5,7 +5,7 @@ def initialize
     end
 
     def extract(association_name, object, local_options, options={})
-      value = @extractor.extract(association_name, object, local_options, options)
+      value = @extractor.extract(association_name, object, local_options, options.except(:default))
       return options[:default] if value.nil?
```

Comment:
Can we just let the `AutoExtractor` deal with the options[:default] stuff? And take it out of here?

Reply:
Yeah I proposed an alternative solution above that let's the AutoExtractor do all the work necessary for defaults.

Reply:
nevermind